docsite/rst/developing_modules.rst

```diff
@@ -490,6 +490,47 @@ Module checklist
 * If you are asking 'how can i have a module execute other modules' ... you want to write a role
 
 
+Windows modules checklist
+`````````````````````````
+* Favour native powershell and .net ways of doing things over calls to COM libraries or calls to native executables which may or may not be present in all versions of windows
+* modules are in powershell (.ps1 files) but the docs reside in same name python file (.py)
+* look at ansible/lib/ansible/module_utils/powershell.ps1 for commmon code, avoid duplication
+* start with::
+
+    #!powershell
+
+then::
+    <GPL header>
+then::
+    # WANT_JSON
+    # POWERSHELL_COMMON
+
+* Arugments:
+    * Try and use state present and state absent like other modules
+    * You need to check that all your mandatory args are present, no boiler plate code yet
```

Comment:
remove ', no boiler plate code yet' and add ' like this::

    If ($params.state) {
        $state = $params.state.ToString().ToLower()
        If (($state -ne 'started') -and ($state -ne 'stopped') -and ($state -ne 'restarted')) {
            Fail-Json $result "state is '$state'; must be 'started', 'stopped', or 'restarted'"
        }
    }